atlasexec/atlas_schema.go

```diff
@@ -18,7 +18,7 @@ type (
 		Context   *RunContext
 		DevURL    string
 
-		Repo        string // Repository name.
+		Name        string // Name of the schema to push.
```

Comment:
```suggestion
		Name        string // Name of the schema (repo) to push to.
```
Or something else to clearify that name is actually repo.